parent/pom.xml

```diff
@@ -131,8 +131,8 @@
     <commons-pool2-version>2.4.2</commons-pool2-version>
     <commons-vfs2-version>2.0</commons-vfs2-version>
     <compress-lzf-version>1.0.3</compress-lzf-version>
-    <consul-client-version>0.12.0</consul-client-version>
-    <consul-client-bundle-version>0.12.0_1</consul-client-bundle-version>
+    <consul-client-version>0.12.2</consul-client-version>
+    <consul-client-bundle-version>0.12.2</consul-client-bundle-version>
```

Comment:
This should be 0.12.2_1 it is an OSGi bundle made by Service Mix (not yet released indeed)